urbanopt_to_alfalfa.ipynb

```diff
@@ -0,0 +1,293 @@
+    "\n",
+    "    - Model Folder: Contains OpenStudio model (.osm file) for each building created using the URBANopt simulation.\n",
+    "    - Measures Folder: Measures to be added while running Alfalfa in the OpenStudio Workflow\n",
+    "    - Weather Folder: Contains EPW weather file \n",
+    "    - workflow.osw file: OpenStudio workflow file\n",
+    "\n",
+    "**Define the following variables before running the code**:\n",
+    "\n",
+    "- `uo_folder` : URBANopt Scenario Directory \n",
+    "- `weather`: Define weather file used in URBANopt project\n",
+    "- `workflow`: OpenStudio workflow file name"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "83836f8d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set URBANopt scenario directory\n",
+    "uo_folder = 'C:/a/baseline_stochastic/run/baseline_scenario_stochastic'\n",
+    "\n",
+    "uo_scenario_folder = uo_folder.split(\"/\")[-1]\n",
+    "uo_scenario_path = os.path.join('./{}'.format(uo_scenario_folder))\n",
+    "Path(uo_scenario_path).mkdir(parents=True, exist_ok=True)\n",
+    "\n",
+    "print(\"Zipped folders saved at {}\".format(uo_scenario_path))\n",
+    "\n",
+    "# Add .epw filename\n",
+    "weather = \"USA_CO_Denver.Intl.AP.725650_TMY3.epw\"\n",
+    "\n",
+    "# Add .osw filename\n",
+    "workflow = \"workflow.osw\"\n",
+    "\n",
+    "for filename in os.listdir(uo_folder):\n",
+    "        \n",
+    "    if os.path.isfile(os.path.join(uo_folder, filename, 'in.osm').replace(\"\\\\\",\"/\")):\n",
+    "        uo_model_filepath = os.path.join(uo_scenario_path, filename)\n",
+    "        Path(uo_model_filepath).mkdir(parents=True, exist_ok=True)\n",
+    "        Path(os.path.join(uo_model_filepath, 'models')).mkdir(parents=True, exist_ok=True)\n",
+    "        Path(os.path.join(uo_model_filepath, 'measures')).mkdir(parents=True, exist_ok=True)\n",
+    "        Path(os.path.join(uo_model_filepath, 'weather')).mkdir(parents=True, exist_ok=True)\n",
+    "\n",
+    "        shutil.copy((os.path.join(uo_folder,filename, 'in.osm')), (os.path.join(uo_model_filepath, 'models', '{}.osm'.format(filename))))\n",
+    "        shutil.copy((os.path.join(uo_folder, '../../weather', weather)), (os.path.join(uo_model_filepath, 'weather')))\n",
+    "        \n",
+    "        osw = {\"seed_file\": \"{}.osm\".format(filename),\n",
+    "           \"weather_file\": \"{}\".format(weather),\n",
+    "           \"run_directory\": \"./run/\",\n",
+    "           \"file_paths\": [\n",
+    "               \"./weather/\",\n",
+    "               \"./models/\"\n",
+    "           ]\n",
+    "          }\n",
+    "        \n",
+    "        f = open(os.path.join(uo_model_filepath, workflow), \"w+\")\n",
+    "        f.write(json.dumps(osw, indent=4))\n",
+    "        f.close()\n",
+    "        \n",
+    "        shutil.make_archive((uo_model_filepath), 'zip', (uo_model_filepath))\n",
```

Comment:
alfalfa-client `0.5.0` automatically zips folders, so this shouldn't be necessary

Reply:
if you want to test on 0.5.0, you can do so with https://alfalfa.nrel.gov I'll be updating cctwin next week.